core-blocks/cover-image/index.js

```diff
@@ -136,21 +137,25 @@ export const settings = {
 							setAttributes( { contentAlign: nextAlign } );
 						} }
 					/>
-					<Toolbar>
-						<MediaUpload
-							onSelect={ onSelectImage }
-							type="image"
-							value={ id }
-							render={ ( { open } ) => (
-								<IconButton
-									className="components-toolbar__control"
-									label={ __( 'Edit image' ) }
-									icon="edit"
-									onClick={ open }
+					{ (
```

Comment:
The `{ (` is unnecessary and can be removed, allowing de-indent of the rest of the render.